index.js

```diff
@@ -396,7 +403,14 @@ class Raspi extends EventEmitter {
           break;
         case PWM_MODE:
         case SERVO_MODE:
-          pinInstance.peripheral = new PWM(normalizedPin);
+          if (pinInstance.isHardwarePwm) {
+            pinInstance.peripheral = new PWM(normalizedPin);
+          } else {
+            pinInstance.peripheral = new SoftPWM({
+              pin: normalizedPin,
+              range: 1000
```

Comment:
For this and the upcoming frequency property, can you create two constants `SOFTWARE_PWM_RANGE` and `SOFTWARE_PWM_FREQUENCY` and put them at the top with the others?